lib/client/vm_api.rb

```diff
@@ -6,7 +6,11 @@ def vm(vm_id, opts={})
     end
 
     def vms(opts={})
-      headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      if opts[:without_details]
+        headers = {:accept => "application/xml"}
+      else
+        headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      end
       path = "/vms"
       path += search_url(opts) unless filtered_api
```

Comment:
@lzap It seems that limitations to oVirt's filtered api prevent any sort of searching or pagination for non-admin users (@oourfali @jhernand - correct me if I'm wrong). I suggest we get this working for admin users first, and see what we can do about non-admins at a later stage.